README.md

```diff
@@ -41,40 +43,43 @@ Minishift requires a hypervisor to run the virtual machine containing OpenShift.
 
-Minishift ships with drivers for VirtualBox and VMware Fusion out of the box. Other drivers
-require manual installation, see [docker machine drivers installation](./docs/docker-machine-drivers.md) for more details.
+Make sure that the hypervisor is installed and enabled on your system before you install Minishift.
 
-Drivers can be selected via the `--vm-driver=xxx` flag of `minishift start` as mentioned in the [Starting Minishift](#starting-minishift) section below. See the [Troubleshooting](/docs/troubleshooting.md#kvm-drivers) guide, in case you encounter any issues.
+**Important:**
 
-**Note:**
-- For most hypervisors, VT-x/AMD-v virtualization must be enabled in the BIOS. For Hyper-V, however,
-it needs to be disabled.
-- We recommend that you use `Virtualbox >= 5.1.12` on Windows to avoid the issue
+- On some operating systems, KVM and xhyve require specific installation and configuration steps.
+For more information, see [docker machine drivers installation](./docs/docker-machine-drivers.md).
+- For most hypervisors, you must enable VT-x/AMD-v virtualization in the BIOS. However, if you want
+to use Hyper-V you must disable this.
+- It is recommended to use `Virtualbox 5.1.12` or later on Windows to avoid the issue
 [Error: getting state for host: machine does not exist](./docs/troubleshooting.md#error-getting-state-for-host-machine-does-not-exist)
 
+If you encounter driver issues, see the [Troubleshooting](/docs/troubleshooting.md) guide.
+
 <a name="installing-minishift"></a>
 ### Installing Minishift
 
-Download the archive matching your host OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it. Copy the contained binary to your preferred
-location and optionally ensure it is added to your _PATH_.
+1. Download the archive for your operating system from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it.
+2. Copy the contents of the directory to your preferred location and add the path to your _PATH_ environment variable.
 
 **Note:**
-- Due to issue [#236](https://github.com/minishift/minishift/issues/236), you need to execute the minishift binary on Windows OS from the drive containing your %USERPROFILE% directory.
-- Automatic update of minishift binary and virtual machine ISO is currently disabled, due to issues
- [#204](https://github.com/minishift/minishift/issues/204),
- [#178](https://github.com/minishift/minishift/issues/178),
- [#112](https://github.com/minishift/minishift/issues/112) and
- [#192](https://github.com/minishift/minishift/issues/192). We will take a comprehensive look at
- these issues in an upcoming release and provide an improved solution for automatic updates.
 
-<a name="installing-via-homebrew"></a>
-#### Installing via Homebrew
+- On Windows operating system, due to issue [#236](https://github.com/minishift/minishift/issues/236), you need to execute the minishift binary
```

Comment:
this comment seems out of context. I might have placed it wrong, but with the old version gone, I am not quite sure what I wanted to comment